pootle/apps/pootle_translationproject/models.py

```diff
@@ -191,11 +191,15 @@ def fullname(self):
 
     @property
     def abs_real_path(self):
-        return absolute_real_path(self.real_path)
+        if self.real_path is not None:
+            return absolute_real_path(self.real_path)
```

Comment:
@unho are you really proposing to go through the code adding `return None` to every function?